Helper/Data.php

```diff
@@ -9,6 +9,8 @@
  */
 class Data extends \Magento\Framework\App\Helper\AbstractHelper
 {
+    const chunkItems = 50;
```

Comment:
why?

Reply:
I am moving this constant to the Data helper since it's loaded almost everywhere and because import model that is currently holding that constant will be removed when other observers (customers, categories, products) are rewritten to account for the new api spec.